### PR TITLE
Begin adding information to exercise anatomy metadata section.

### PR DESCRIPTION
**Pull request by Insti**
Basic information taken from: https://github.com/exercism/problem-specifications/blob/master/README.md#problem-metadata

This provides a starting point for consolidating documentation about the metadata files.

### CODE REVIEW

File: language-tracks/exercises/anatomy/README.md [hunk 1]
```diff
@@ -14,6 +14,47 @@ TODO
...
+
+Format, valid/required/default keys, example.
+
+### Metadata.yml
+
+Format, valid/required/default keys, example.
+
+### canonical-data.json
+
+Has it's own documentation. See: TODO
```
Comment (petertseng) [inline language-tracks/exercises/anatomy/README.md]: it's -> its

File: language-tracks/exercises/anatomy/README.md [hunk 2]
```diff
@@ -14,6 +14,47 @@ TODO
...
+There are three metadata files:
+
+* `description.md` - the basic problem description
+* `metadata.yml` - additional information about the problem, such as where it came from
+* `canonical-data.json` (optional) - standardized test inputs and outputs that can be used to implement the problem
+
+
+### Description.md
+
+Format, valid/required/default keys, example.
```
Comment (petertseng) [inline language-tracks/exercises/anatomy/README.md]: keys doesn't seem like the right word here since the md file doesn't necessarily have key/value pairs

**Comment by Insti** [comment]
Other relevant metadata information can be found here: https://github.com/exercism/docs/blob/master/you-can-help/make-up-new-exercises.md

And possibly should be moved into this document by a future PR.

**Review by petertseng — COMMENTED** [review]
Because of the following reasons:

1. This is under directory language-tracks
2. Test suites are mentioned
3. reference solution is mentioned

That is making me think these are the files that should show up in each language track. Now, canonical-data would not appear in each individual language track. The other two files might, but they would go in `.meta`. I think there is some inconsistency here, but it could work if the inconsistency were resolved somehow. I would want not to duplicate the information about metadata.yml that already exist in https://github.com/exercism/docs/blob/master/you-can-help/make-up-new-exercises.md#problem-specification, so I am thinking in the final state one would link to the other. No opinion on which way the link should run.

**Comment by Insti** [comment]
>  I think there is some inconsistency here,

I think there are some conceptual issues with the way this documentation is currently laid out.

To me, exercise anatomy covers both what is in the track AND what is in problem-specifications. 
Perhaps these should be addressed in separate documents within this directory and cross linked as appropriate.

**Comment by Insti** [comment]
See also: https://github.com/exercism/docs/issues/100 where I try to gain clarity on document locations.

**Comment by Insti** [comment]
Closing this, as it was written for version 1.
Some of the information may still be relevant for version 2 but I don't feel I know enough to comment authoritatively.

Feel free to use what was here if it is still relevant and you want to make a new PR.

**Pull request closed without merging**